python/ray/tune/trial_runner.py

```diff
@@ -9,6 +9,7 @@
 import logging
 import os
 import re
+from tabulate import tabulate
```

Comment:
this is an external dependency; maybe we can add this to the ray dependencies in `setup.py`?